### DISCUSSION TITLE
UGS on MAC port for Arduino Uno controller not appearing

### DISCUSSION THREAD

**Discussion by culubunny** [winder/Universal-G-Code-Sender#1790] [Help] [answered]
When running UGS Platform on Mac, Arduino controller port not appearing in UGS. Other g code senders have no problem. thanks 

MacBook Pro mid 2012 10.15.7

**Comment by breiler** [comment, marked as answer]
When using the terminal, what is listed when you type these commands.

This will list connected USB serial devices:
```ls /dev/tty.*```

Disconnect the device and connect it again and type this command. It will show the latest 100 diagnostic messages:
```sudo dmesg | tail -n 100```

What other senders have you tried?

**Reply by culubunny** [reply]
hi and thanks

followed first set of instructions and notice tty suffix which appeared in terminal and helped me understand what to look for in ugs. i was successful. thank you.
